components/src/LiveCode/liveCodeScope.tsx

```diff
@@ -0,0 +1,4 @@
+import Guidelines from '../Guidelines'
+import StatusTag from '../StatusTag'
+
+export default { Guidelines, StatusTag }
```

Comment:
Maybe the default scope should be empty and adapt the examples on storybook :thinking: what do you guys think?

Reply:
This should be `{}` by default! :smile: